provider/lib/health.js

```diff
@@ -1,11 +1,39 @@
-module.exports = function(providerUtils) {
+var si = require('systeminformation');
 
-  // Health Endpoint
-  this.endPoint = '/health';
+module.exports = function(utils) {
 
-  // Health Logic
-  this.health = function (req, res) {
-      res.send({triggerCount: Object.keys(providerUtils.triggers).length});
-  };
+    // Health Endpoint
+    this.endPoint = '/health';
+    var stats = {triggerCount: Object.keys(utils.triggers).length};
+
+    // Health Logic
+    this.health = function (req, res) {
```

Comment:
Each of these `si` calls are async and potentially time consuming. They also do not depend on each other... for example, the call to `si.currentLoad()` doesn't actually depend on the result of calling `si.mem()`. I suggest queueing these up into a Promise array then using `Promise.all()` to collect the results. This will help minimize the performance impact of gathering all these hardware stats.